README.md

```diff
@@ -34,13 +34,15 @@ For this guide we assume the following directory structure:
 ├── build.gradle
 ├── gradle
 ├── greeter.api
+├── greeter.demo
 ├── greeter.provider
 ├── greeter.provider.test
 ├── greeter.runner
 └── settings.gradle
 ```
 
 * greeter.api: Exports an interface
+* greeter.demo: Applications that can be started with `ModularJavaExec` tasks
```

Comment:
Can we rename it to something more descriptive?

Reply:
I think`greeter.javaexec` would be good. My guess is the different examples will only grow, so it would be good to start naming them to show what they are doing.

Reply:
I renamed it and provided extra jvm arguments.